code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -8,10 +8,14 @@
 	icon_dead = "forsakenmurdererdead"
 	portrait = "forsaken_murderer"
 	del_on_death = FALSE
-	/* Originally was 270. Fragment health is 800 with a original game health of 230 so techically forsaken
-		murderer has more health than fragment? Ill round the numbers to 600 since 270 can be rounded to 300 and doubled.
-		I was later told to make it 1100. We really dont have a Lobotomy Corp to LC13 health conversion calculator.*/
-	// Turns out that 1100 is not enough. - Kirie/Kitsunemitsu
+	/**
+	 * Originally was 270.
```

Comment:
As the coder for Forsaken i do approve of this reformatting of the comments to make them more readable.